pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -238,7 +238,13 @@ func (a *ApplicationBackupController) handle(ctx context.Context, backup *stork_
 			err.Error())
 		return nil
 	}
-
+	// Listing all resource types
+	if len(backup.Spec.ResourceTypes) != 0 && len(backup.Spec.IncludeResources) != 0 {
+		// Currently this is not supported, error out
+		errMsg := "cannot have ResourceType and IncludeResources together"
```

Comment:
nit: `cannot set both ResourceType and IncludeResources in ApplicationBackup request`